### PR TITLE
Fixing a syntax error in the apps.json

### PR DESCRIPTION
**Pull request by jmprieur**

### CODE REVIEW

**Comment by jmprieur** [comment]
@archieag

File: AppCreationScripts/apps.json [hunk 1]
```diff
@@ -35,7 +35,6 @@
             "appId": "4a1aa1d5-c567-49d0-ad0b-cd957a47f842",
             "tenantId": "common",
             "authorityEndpointHost": "https://login.microsoftonline.com/"
-          }
```
Comment (archieag) [inline AppCreationScripts/apps.json]: Still invalid. Seems like you need to add a closing curly bracket after line #36.

Comment (archieag) [reply AppCreationScripts/apps.json]: And not delete the one at Ln #38.

Comment (jmprieur) [reply AppCreationScripts/apps.json]: Thanks @archieag 
fixed
Learning of the day: don't try to fix something from the online UI between 2 meetings :)

**Comment by archieag** [comment]
lgtm :) @jmprieur

**Pull request merged**